scalar_grpc_web_pb.js

```diff
@@ -361,5 +308,733 @@ proto.rpc.LedgerPromiseClient.prototype.validateLedger =
+ *     The XHR Node Readable Stream
+ */
+proto.rpc.LedgerPrivilegedClient.prototype.registerCert =
+    function(request, metadata, callback) {
+  return this.client_.rpcCall(this.hostname_ +
+      '/rpc.LedgerPrivileged/RegisterCert',
+      request,
+      metadata || {},
+      methodInfo_LedgerPrivileged_RegisterCert,
+      callback);
+};
+
+
+/**
+ * @param {!proto.rpc.CertificateRegistrationRequest} request The
+ *     request proto
+ * @param {?Object<string, string>} metadata User defined
+ *     call metadata
+ * @return {!Promise<!proto.rpc.LedgerServiceResponse>}
+ *     A native promise that resolves to the response
+ */
+proto.rpc.LedgerPrivilegedPromiseClient.prototype.registerCert =
+    function(request, metadata) {
+  return this.client_.unaryCall(this.hostname_ +
+      '/rpc.LedgerPrivileged/RegisterCert',
+      request,
+      metadata || {},
+      methodInfo_LedgerPrivileged_RegisterCert);
+};
+
+
+/**
+ * @const
+ * @type {!grpc.web.AbstractClientBase.MethodInfo<
+ *   !proto.rpc.FunctionRegistrationRequest,
+ *   !proto.rpc.LedgerServiceResponse>}
+ */
+const methodInfo_LedgerPrivileged_RegisterFunction = new grpc.web.AbstractClientBase.MethodInfo(
+  proto.rpc.LedgerServiceResponse,
+  /** @param {!proto.rpc.FunctionRegistrationRequest} request */
+  function(request) {
+    return request.serializeBinary();
+  },
+  proto.rpc.LedgerServiceResponse.deserializeBinary
+);
+
+
+/**
+ * @param {!proto.rpc.FunctionRegistrationRequest} request The
+ *     request proto
+ * @param {?Object<string, string>} metadata User defined
+ *     call metadata
+ * @param {function(?grpc.web.Error, ?proto.rpc.LedgerServiceResponse)}
+ *     callback The callback function(error, response)
+ * @return {!grpc.web.ClientReadableStream<!proto.rpc.LedgerServiceResponse>|undefined}
+ *     The XHR Node Readable Stream
+ */
+proto.rpc.LedgerPrivilegedClient.prototype.registerFunction =
+    function(request, metadata, callback) {
+  return this.client_.rpcCall(this.hostname_ +
+      '/rpc.LedgerPrivileged/RegisterFunction',
+      request,
+      metadata || {},
+      methodInfo_LedgerPrivileged_RegisterFunction,
+      callback);
+};
+
+
+/**
+ * @param {!proto.rpc.FunctionRegistrationRequest} request The
+ *     request proto
+ * @param {?Object<string, string>} metadata User defined
+ *     call metadata
+ * @return {!Promise<!proto.rpc.LedgerServiceResponse>}
+ *     A native promise that resolves to the response
+ */
+proto.rpc.LedgerPrivilegedPromiseClient.prototype.registerFunction =
+    function(request, metadata) {
+  return this.client_.unaryCall(this.hostname_ +
+      '/rpc.LedgerPrivileged/RegisterFunction',
+      request,
+      metadata || {},
+      methodInfo_LedgerPrivileged_RegisterFunction);
+};
+
+
+/**
+ * @param {string} hostname
+ * @param {?Object} credentials
+ * @param {?Object} options
+ * @constructor
+ * @struct
+ * @final
+ */
+proto.rpc.AdminClient =
```

Comment:
I'm really sorry. What SDK should support now are only `Ledger`, `LedgerPrivileged` and `ProofRegistry`.
So we don't need other services at the moment.
Sorry for saying this now.

Reply:
No problem. I will remove these services from proto files and then regenerate stub files.